### PR TITLE
WPC-788: Subscription edit as separate view

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@antkiewiczk is this part of specific task ?

**Review by Itrulia — DISMISSED** [review]

File: apps/editor/src/app/routes/userEditView.tsx [hunk 1]
```diff
@@ -697,7 +697,7 @@ function UserEditView() {
             <Col xs={12}>
               <Grid fluid>
                 <RPanel bordered header={t('userCreateOrEditView.subscriptionsHeader')}>
-                  <UserSubscriptionsList subscriptions={user?.subscriptions} />
+                  <UserSubscriptionsList subscriptions={user?.subscriptions} userId={user?.id} />
```
Comment (tomaszdurka) [inline apps/editor/src/app/routes/userEditView.tsx:700]: Render whole Subscription Column only if the user exists. Get rid of `user?.*`

File: apps/editor/src/app/routes/subscriptionList.tsx [hunk 2]
```diff
@@ -301,7 +323,7 @@ function SubscriptionList() {
           setEditModalOpen(false)
           navigate('/subscriptions')
         }}>
-        <SubscriptionEditPanel
+        <SubscriptionEditView
```
Comment (michael-scheurer) [inline apps/editor/src/app/routes/subscriptionList.tsx:326]: This drawer can be removed. Instead you need to route the user.

**Review by michael-scheurer — CHANGES_REQUESTED** [review]
Good work! See one comment in the code and the screenshot below. Also you might align the deactivate button. I would move it to the top right part. See for example how we did it on the comment single view.

image

**Review by Itrulia — APPROVED** [review]

**Review by penk85 — APPROVED** [review]

**Review by michael-scheurer — APPROVED** [review]

**Pull request merged**